src/coprocessor/endpoint.rs

```diff
@@ -154,6 +154,7 @@ pub struct Host {
     running_task_count: Arc<AtomicUsize>,
     batch_row_limit: usize,
     stream_batch_row_limit: usize,
+    stream_channel_size: usize,
```

Comment:
This looks like a network related configuration, it feels weird to put it inside `Host`.